packages/dd-trace/src/appsec/iast/analyzers/ssrf-analyzer.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const InjectionAnalyzer = require('./injection-analyzer')
+const { SSRF } = require('../vulnerabilities')
+
+class SSRFAnalyzer extends InjectionAnalyzer {
+  constructor () {
+    super(SSRF)
+
+    this.addSub('apm:http:client:request:start', ({ originalArgs }) => {
+      if (originalArgs.wholeUrl) {
```

Comment:
if `wholeUrl` has value, analyzer skips the `options.host` analysis. 
Is that correct or do you want to analyze `options.host` always, if not undefined?

Reply:
I made it at this way, because the host parameter is ignored when the whole url parameter is set.

Reply:
In this case, if wholeUrl is an empty string, we will analyze `options.host`, because `if ('') {` is false.
I can change it to check like:

```suggestion
      if (originalArgs.wholeUrl || typeof originalArgs.wholeUrl === 'string') {
```

wdyt?

Reply:
yes, but empty string is falsy. 
`if (originalArgs.wholeUrl) {` is false when  `wholeUrl === ''`, but `if (typeof originalArgs.wholeUrl === 'string')` is true